### PR TITLE
Prestashop installer

### PR DESCRIPTION
**Pull request by Ekman**
Prestashop installer for composer modules and themes.

### CODE REVIEW

**Comment by shama** [comment]
Thanks, could you add tests too?

**Comment by Ekman** [comment]
Created a test. Not sure it is entirely correct, let me know!

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 vendor/
 composer.lock
 .idea/
+/nbproject/
```
Comment (shama) [inline .gitignore]: Mind removing this?

**Comment by shama** [comment]
Sorry I should have be more specific, take a look at https://github.com/composer/installers/blob/master/tests/Composer/Installers/Test/InstallerTest.php#L144

We just need a couple lines that tests support for the installer and support for each of the types. That way we explicitly know what it is supposed to resolve to.

Usually I only create individual test cases for installers that perform custom functionality outside of the base installer.

**Comment by Ekman** [comment]
That makes sense! Fixed the tests and removed NetBeans from gitignore

**Comment by shama** [comment]
Thanks!

**Pull request merged**